sql/moz-fx-data-marketing-prod/ga_derived/www_site_events_metrics_v2/schema.yaml

```diff
@@ -0,0 +1,93 @@
+fields:
+- mode: NULLABLE
+  name: date
+  type: DATE
+  description: Date of the visit
+- mode: NULLABLE
+  name: event_category
+  type: STRING
+  description: Event Category - Since event_category doesn't exist in GA4, this is event_name
+- mode: NULLABLE
+  name: event_action
+  type: STRING
+  description: Event Action - Since event_action doesn't exist in GA4, this is event_name
+- mode: NULLABLE
+  name: event_label
+  type: STRING
+  description: Event Label - Since event_label doesn't exist in GA4, this is event_name
+- mode: NULLABLE
+  name: page_path
+  type: STRING
+  description: Page Path
+- mode: NULLABLE
+  name: locale
+  type: STRING
+  description: Locale
+- mode: NULLABLE
+  name: page_level_1
+  type: STRING
+  description: Page Level 1
+- mode: NULLABLE
+  name: page_level_2
+  type: STRING
+  description: Page Level 2
+- mode: NULLABLE
+  name: page_level_3
+  type: STRING
+  description: Page Level 3
+- mode: NULLABLE
+  name: page_level_4
+  type: STRING
+  description: Page Level 4
+- mode: NULLABLE
+  name: page_level_5
+  type: STRING
+  description: Page Level 5
+- mode: NULLABLE
+  name: device_category
+  type: STRING
+  description: Device Category - The device category the visitor used to visit the site
+- mode: NULLABLE
+  name: operating_system
+  type: STRING
+  description: Operating System - The operating system the visitor used to visit the site
+- mode: NULLABLE
+  name: language
+  type: STRING
+  description: Language
```

Comment:
Shall we add the difference between this column and locale?

Reply:
Good idea, will do

Reply:
Just pushed the updates